healthtools/tests/test_scrapers.py

```diff
@@ -59,6 +77,24 @@ def test_health_facilities_scraper_uploads_to_elasticsearch(self):
             response = self.health_facilities_scraper.upload(json.loads(data))
             self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
 
+    def test_nhif_accredited_scraper_uploads_to_elasticsearch(self):
+        with open(TEST_DIR + "/dummy_files/nhif_accredited_facilities.json", "r") as my_file:
+            data = my_file.read()
+            response = self.nhif_accredited_scraper.upload_data(json.loads(data))
+            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+
+    def test_nhif_inpatient_scraper_uploads_to_elasticsearch(self):
+        with open(TEST_DIR + "/dummy_files/nhif_inpatient_facilities.json", "r") as my_file:
+            data = my_file.read()
+            response = self.nhif_inpatient_scraper.upload_data(json.loads(data))
+            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+
+    def test_nhif_outpatient_scraper_uploads_to_elasticsearch(self):
+        with open(TEST_DIR + "/dummy_files/nhif_outpatient_facilities.json", "r") as my_file:
+            data = my_file.read()
+            response = self.nhif_outpatient_scraper.upload_data(json.loads(data))
+            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
```

Comment:
Let's remove "facilities" in the names. It's redundant - "Inpatient", "Outpatient" already talks about a facility.